config.yaml

```diff
@@ -31,9 +31,8 @@ metriccache.check.ids: []
 
 plugin.eventlog.eventlog_url: 'https://eventlog.example.com/'
 
-plugin.history.kairosdb_host: 'localhost'
-plugin.history.kairosdb_port: 8083
-plugin.history.kairosdb_history_enabled: False
+plugin.history.url: 'http://localhost:8083'
+plugin.history.kairosdb_history_enabled: true
```

Comment:
Main intent with the week was History is used for week in week comparisons.
Thus give me the 5min last week and compare.

On Thursday, 16 June 2016, Henning Jacobs <notifications@github.com> wrote:

> In config.yaml
> <https://github.com/zalando-zmon/zmon-worker/pull/70#discussion_r67349382>
> :
>
> > @@ -31,9 +31,8 @@ metriccache.check.ids: []
> >
> >  plugin.eventlog.eventlog_url: 'https://eventlog.example.com/'
> >
> > -plugin.history.kairosdb_host: 'localhost'
> > -plugin.history.kairosdb_port: 8083
> > -plugin.history.kairosdb_history_enabled: False
> > +plugin.history.url: 'http://localhost:8083'
> > +plugin.history.kairosdb_history_enabled: true
>
> cam we rename to plugin.history.enabled? looks inconsistent right now
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zalando-zmon/zmon-worker/pull/70/files/7b44814fe927589cd2426a4fb0c505b04e1af6bf#r67349382>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABYGQzXR8g-apAfmpeHyBIqOFv067Wf0ks5qMVdtgaJpZM4I3OmV>
> .
>